src/authentication/controller/auth.controller.ts

```diff
@@ -0,0 +1,15 @@
+import { Controller, Get, Req } from '@nestjs/common';
+import { AuthService } from '../service/auth.service';
+import { ApiTags } from '@nestjs/swagger';
+import { ReqUser } from '../dto/user.dto/users.dto';
+
+@Controller('userAuth')
+@ApiTags('Auth')
+export class AuthController {
+  constructor(private authService: AuthService) {}
+  @Get('verifyUser')
```

Comment:
Not kept rest standarts

Reply:
I'm not entirely sure if I understood you correctly. I've changed the names of requests and functions, and I hope that's what you meant.